src/context.rs

```diff
@@ -622,7 +622,7 @@ impl Context {
 
     pub fn get_font_face(&self) -> FontFace {
         unsafe {
-            FontFace(ffi::cairo_get_font_face(self.0))
+            from_glib_none(ffi::cairo_get_font_face(self.0))
```

Comment:
Looks like `ScaledFont` also has [the exact same bug](https://github.com/johncf/cairo/blob/f87f713c/src/context.rs#L637).